### PR TITLE
[WT-263]  - Change way of reporting files and folders with same name uploaded

### PR DESCRIPTION
**Pull request by CandelR**
- Added logic and modal to handle name collision when uploading or moving elements

### CODE REVIEW

**Review by sg-gs — APPROVED** [review]

**Comment by sg-gs** [comment]
I request the @aboredvaro review this behavior to ensure all is fine.

**Review by aboredvaro — CHANGES_REQUESTED** [review]
Issues found:
- Must use the correct dialog from [name collision dialog branch](https://github.com/internxt/drive-web/tree/origin/feature/WT-263-name-collision-dialog) to ensure right design implementation and usability standards.
- When user selects "Keep both", everything goes as expected, but an error "Folder not found" pops up.
- When user selects "Replace current item" the folder corrupts, the item that is being replaced is moved to trash (this is ok), but then the new item that is uploaded never gets uploaded (a bunch of errors appear) and from that moment user can't upload anything to that parent folder. [See video for example](https://send.internxt.com/download/0c9cdec7-f21e-4660-958a-6e25f92293c9?code=6b975ecda788889faac28e2d12ecba260417a0f2b0dea3cc76a71302d098a20b).

**Comment by CandelR** [comment]
> Issues found:
> 
> * Must use the correct dialog from [name collision dialog branch](https://github.com/internxt/drive-web/tree/origin/feature/WT-263-name-collision-dialog) to ensure right design implementation and usability standards.
> * When user selects "Keep both", everything goes as expected, but an error "Folder not found" pops up.
> * When user selects "Replace current item" the folder corrupts, the item that is being replaced is moved to trash (this is ok), but then the new item that is uploaded never gets uploaded (a bunch of errors appear) and from that moment user can't upload anything to that parent folder. [See video for example](https://send.internxt.com/download/0c9cdec7-f21e-4660-958a-6e25f92293c9?code=6b975ecda788889faac28e2d12ecba260417a0f2b0dea3cc76a71302d098a20b).

@aboredvaro 
- Okay, I will check the branch dialogue and replace the current one.

- Regarding the errors on the "Keep both" and "Replace current item" it is a matter of Vercel environment variables not being correct. I check it also because apparently you can't upload files in the Vercel Preview and it probably has to do with the same thing, a configuration error in the Preview.

File: src/app/drive/components/DriveExplorer/DriveExplorer.tsx [hunk 1]
```diff
@@ -300,6 +353,9 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
                       <FolderSimplePlus size={20} />
                       <p className="ml-3">{i18n.get('actions.upload.folder')}</p>
                     </MenuItem>,
+                    <MenuItem onClick={() => dispatch(uiActions.setIsNameCollisionDialogOpen(true))}>
+                      <p>Test name collision</p>
```
Comment (aboredvaro) [inline src/app/drive/components/DriveExplorer/DriveExplorer.tsx]: Remove this testing menu item
Screenshot 2023-01-18 at 16 57 33

**Review by aboredvaro — CHANGES_REQUESTED** [review]

**Review by aboredvaro — APPROVED** [review]

**Pull request merged**